src/Combinators/altGuard.ts

```diff
@@ -0,0 +1,16 @@
+import {ReasonGuard} from '../ReasonGuard';
+import {notGuard} from './notGuard';
+import {buildNegatable} from '../NegatableGuard';
+
+export const altGuard = <LEFT, RIGHT>(left: ReasonGuard<LEFT|RIGHT, LEFT>) => buildNegatable(
```

Comment:
This feels like it deserves an explanatory comment -- I eventually figured out what it's doing, but it's not obvious from the way it's written